prometheus_client/context_managers.py

```diff
@@ -1,21 +1,30 @@
 from timeit import default_timer
+from types import TracebackType
+from typing import (
+    Any, Callable, Literal, Optional, Type, TYPE_CHECKING, TypeVar,
+)
 
 from .decorator import decorate
 
+if TYPE_CHECKING:
+    from . import Counter
+    F = TypeVar("F", bound=Callable[..., Any])
+
 
 class ExceptionCounter:
-    def __init__(self, counter, exception):
+    def __init__(self, counter: "Counter", exception: Type[BaseException]) -> None:
```

Comment:
:thinking: Is there any downside to importing `Counter` all of the time and keeping this type without quotes similar to other cases? To use this function the user will have to import Counter anyway.

Reply:
Ohh wait, nevermind. Circular import, resolving this conversation.